Latest version of the Library has compilation issues

## 🐛 Bug Report
I keep getting a bunch of `package android.xx not found` errors when I try to run my React Native App.

Here is the error log:

```
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:3: error: package android.annotation does not exist
import android.annotation.TargetApi;
                         ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:4: error: package android.content does not exist
import android.content.Intent;
                      ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:5: error: package android.content.res does not exist
import android.content.res.Configuration;
                          ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:6: error: package android.graphics does not exist
import android.graphics.Color;
                       ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:7: error: package android.os does not exist
import android.os.Build;
                 ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:8: error: package android.os does not exist
import android.os.Bundle;
                 ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:9: error: package android.view does not exist
import android.view.KeyEvent;
                   ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:10: error: package android.view does not exist
import android.view.View;
                   ^
node_modules\react-native-navigation\lib\android\app\src\main\java\com\reactnativenavigation\NavigationActivity.java:29: error: cannot access ViewGroup
public class NavigationActivity extends AppCompatActivity implements DefaultHardwareBackBtnHandler, PermissionAwareActivity, JsDevReloadHandler.ReloadListener {
       ^
  class file for android.view.ViewGroup not found
9 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-navigation:compileReactNative63DebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 17s

    at makeError (node_modules\execa\index.js:174:9)
    at node_modules\execa\index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runOnAllDevices (node_modules\@react-native-community\cli-platform-android\build\commands\runAndroid\runOnAllDevices.js:94:5)
    at async Command.handleAction (node_modules\@react-native-community\cli\build\index.js:186:9)
info Run CLI with --verbose flag for more details.`
```

### Have you read the [Contributing Guidelines on issues](https://wix.github.io/react-native-navigation/docs/meta-contributing/)?

Yes

## To Reproduce

(Write your steps here:)

1.  Create a clean project using React Native CLI - `npx react-native init Issue`
2. Install the react-native-navigation package with yarn - `yarn add react-native-navigation`
3. Run `npx rnn-link`
4. Start the application using `npx react-native run-android`

## Expected behavior
I expected the application to compile and run.

## Actual Behavior
Multiple compilation errors.

## Your Environment
- React Native Navigation version: 7.30.1
- React Native version: 0.64.2
- Platform(s) (iOS, Android, or both?): Android
- Device info (Simulator/Device? OS version? Debug/Release?): Simulator, Android 11, Debug

## Reproducible Demo
N/A

### Are you willing to resolve this issue by submitting a Pull Request?
 - ✖️  &nbsp; Yes, I have the time, and I know how to start.
 - ✅  &nbsp; Yes, I have the time, but I don't know how to start. I would need guidance.
 - ✖️  &nbsp; No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

I also encountered a similar problem while updating the library to the current version (7.30.1).

```
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/Orientation.java:3: error: package android.content.pm does not exist
import android.content.pm.ActivityInfo;
                         ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/OrientationOptions.java:3: error: package org.json does not exist
import org.json.JSONArray;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/OrientationOptions.java:4: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/OrientationOptions.java:15: error: cannot find symbol
    public static OrientationOptions parse(JSONObject json) {
                                           ^
  symbol:   class JSONObject
  location: class OrientationOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/FractionParser.java:6: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/FractionParser.java:9: error: cannot find symbol
    public static Fraction parse(JSONObject json, String fraction) {
                                 ^
  symbol:   class JSONObject
  location: class FractionParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/LayoutNodeParser.java:7: error: package org.json does not exist
import org.json.JSONArray;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/LayoutNodeParser.java:8: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/LayoutNodeParser.java:16: error: cannot find symbol
        public static LayoutNode parse(JSONObject layoutTree) {
                                       ^
  symbol:   class JSONObject
  location: class LayoutNodeParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutNode.java:3: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/LayoutNodeParser.java:25: error: cannot find symbol
        private static List<LayoutNode> parseChildren(JSONObject layoutTree) {
                                                      ^
  symbol:   class JSONObject
  location: class LayoutNodeParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/LayoutNodeParser.java:36: error: cannot find symbol
        private static JSONObject parseData(JSONObject layoutTree) {
                                            ^
  symbol:   class JSONObject
  location: class LayoutNodeParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/LayoutNodeParser.java:36: error: cannot find symbol
        private static JSONObject parseData(JSONObject layoutTree) {
                       ^
  symbol:   class JSONObject
  location: class LayoutNodeParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutNode.java:23: error: cannot find symbol
        public final JSONObject data;
                     ^
  symbol:   class JSONObject
  location: class LayoutNode
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutNode.java:31: error: cannot find symbol
        public LayoutNode(String id, Type type, JSONObject data, List<LayoutNode> children) {
                                                ^
  symbol:   class JSONObject
  location: class LayoutNode
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutNode.java:38: error: cannot find symbol
    JSONObject getOptions() {
    ^
  symbol:   class JSONObject
  location: class LayoutNode
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/IconParser.java:6: error: package org.json does not exist
import org.json.JSONException;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/IconParser.java:7: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/IconParser.java:12: error: cannot find symbol
    public static Text parse(@Nullable JSONObject json, String key) {
                                       ^
  symbol:   class JSONObject
  location: class IconParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:4: error: package android.animation does not exist
import android.animation.TimeInterpolator;
                        ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:5: error: package android.view.animation does not exist
import android.view.animation.AccelerateDecelerateInterpolator;
                             ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:6: error: package android.view.animation does not exist
import android.view.animation.AccelerateInterpolator;
                             ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:7: error: package android.view.animation does not exist
import android.view.animation.DecelerateInterpolator;
                             ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:8: error: package android.view.animation does not exist
import android.view.animation.LinearInterpolator;
                             ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:9: error: package android.view.animation does not exist
import android.view.animation.OvershootInterpolator;
                             ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:14: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:19: error: cannot find symbol
    public static TimeInterpolator parse(JSONObject json) {
                                         ^
  symbol:   class JSONObject
  location: class InterpolationParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/InterpolationParser.java:19: error: cannot find symbol
    public static TimeInterpolator parse(JSONObject json) {
                  ^
  symbol:   class TimeInterpolator
  location: class InterpolationParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/DensityPixelParser.java:6: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/DensityPixelParser.java:9: error: cannot find symbol
    public static DensityPixel parse(JSONObject json, String number) {
                                     ^
  symbol:   class JSONObject
  location: class DensityPixelParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/params/DensityPixel.java:3: error: package android.content.res does not exist
import android.content.res.Resources;
                          ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/TextParser.java:7: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/TextParser.java:12: error: cannot find symbol
    public static Text parse(@Nullable JSONObject json, String text, String defaultValue) {
                                       ^
  symbol:   class JSONObject
  location: class TextParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/TextParser.java:16: error: cannot find symbol
    public static Text parse(@Nullable JSONObject json, String text) {
                                       ^
  symbol:   class JSONObject
  location: class TextParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:12: error: package org.json does not exist
import org.json.JSONArray;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:13: error: package org.json does not exist
import org.json.JSONException;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:14: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:19: error: cannot find symbol
    public JSONObject parse(ReadableMap map) {
           ^
  symbol:   class JSONObject
  location: class JSONParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:51: error: cannot find symbol
    public JSONArray parse(ReadableArray arr) {
           ^
  symbol:   class JSONArray
  location: class JSONParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:101: error: cannot find symbol
    public static WritableMap convert(JSONObject jsonObject) {
                                      ^
  symbol:   class JSONObject
  location: class JSONParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/JSONParser.java:128: error: cannot find symbol
    public static WritableArray convert(JSONArray jsonArray) {
                                        ^
  symbol:   class JSONArray
  location: class JSONParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/FloatParser.java:6: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/FloatParser.java:9: error: cannot find symbol
    public static FloatParam parse(JSONObject json, String number) {
                                   ^
  symbol:   class JSONObject
  location: class FloatParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/BoolParser.java:6: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/BoolParser.java:13: error: cannot find symbol
    public static Bool parse(JSONObject json, String key) {
                             ^
  symbol:   class JSONObject
  location: class BoolParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/BoolParser.java:17: error: cannot find symbol
    public static Bool parseFirst(JSONObject json, String... keys) {
                                  ^
  symbol:   class JSONObject
  location: class BoolParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/NumberParser.java:6: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/NumberParser.java:9: error: cannot find symbol
    public static Number parse(JSONObject json, String number) {
                               ^
  symbol:   class JSONObject
  location: class NumberParser
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/TopTabOptions.java:3: error: package android.graphics does not exist
import android.graphics.Typeface;
                       ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/TopTabOptions.java:11: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/TopTabOptions.java:15: error: cannot find symbol
    @Nullable public Typeface fontFamily;
                     ^
  symbol:   class Typeface
  location: class TopTabOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/TopTabOptions.java:18: error: cannot find symbol
    public static TopTabOptions parse(TypefaceLoader typefaceManager, JSONObject json) {
                                                                      ^
  symbol:   class JSONObject
  location: class TopTabOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/NavigationBarOptions.java:3: error: package android.content does not exist
import android.content.Context;
                      ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/NavigationBarOptions.java:11: error: package org.json does not exist
import org.json.JSONObject;
               ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/NavigationBarOptions.java:14: error: cannot find symbol
    public static NavigationBarOptions parse(Context context, JSONObject json) {
                                             ^
  symbol:   class Context
  location: class NavigationBarOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/NavigationBarOptions.java:14: error: cannot find symbol
    public static NavigationBarOptions parse(Context context, JSONObject json) {
                                                              ^
  symbol:   class JSONObject
  location: class NavigationBarOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/BottomTabsOptions.java:25: error: cannot find symbol
    public static BottomTabsOptions parse(Context context, JSONObject json) {
                                          ^
  symbol:   class Context
  location: class BottomTabsOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/BottomTabsOptions.java:25: error: cannot find symbol
    public static BottomTabsOptions parse(Context context, JSONObject json) {
                                                           ^
  symbol:   class JSONObject
  location: class BottomTabsOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/BottomTabOptions.java:24: error: cannot find symbol
    public static BottomTabOptions parse(Context context, TypefaceLoader typefaceManager, JSONObject json) {
                                         ^
  symbol:   class Context
  location: class BottomTabOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/BottomTabOptions.java:24: error: cannot find symbol
    public static BottomTabOptions parse(Context context, TypefaceLoader typefaceManager, JSONObject json) {
                                                                                          ^
  symbol:   class JSONObject
  location: class BottomTabOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/DotIndicatorOptions.java:19: error: cannot find symbol
    public static DotIndicatorOptions parse(Context context, @Nullable JSONObject json) {
                                            ^
  symbol:   class Context
  location: class DotIndicatorOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/DotIndicatorOptions.java:19: error: cannot find symbol
    public static DotIndicatorOptions parse(Context context, @Nullable JSONObject json) {
                                                                       ^
  symbol:   class JSONObject
  location: class DotIndicatorOptions
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutFactory.java:3: error: package android.app does not exist
import android.app.Activity;
                  ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutFactory.java:53: error: cannot find symbol
        private Activity activity;
                ^
  symbol:   class Activity
  location: class LayoutFactory
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/react/events/EventEmitter.java:3: error: package android.util does not exist
import android.util.Log;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutFactory.java:70: error: cannot find symbol
        public void init(Activity activity, EventEmitter eventEmitter, ChildControllersRegistry childRegistry, Map<String, ExternalComponentCreator> externalComponentCreators) {
                         ^
  symbol:   class Activity
  location: class LayoutFactory
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:6: error: package android.app does not exist
import android.app.Activity;
                  ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:7: error: package android.content.res does not exist
import android.content.res.Configuration;
                          ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:8: error: package android.view does not exist
import android.view.View;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:9: error: package android.view does not exist
import android.view.ViewGroup;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:10: error: package android.view does not exist
import android.view.ViewManager;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:11: error: package android.view does not exist
import android.view.ViewTreeObserver;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:37: error: package ViewTreeObserver does not exist
public abstract class ViewController<T extends ViewGroup> implements ViewTreeObserver.OnGlobalLayoutListener,
                                                                                     ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:38: error: package ViewGroup does not exist
        ViewGroup.OnHierarchyChangeListener,
                 ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:37: error: cannot find symbol
public abstract class ViewController<T extends ViewGroup> implements ViewTreeObserver.OnGlobalLayoutListener,
                                               ^
  symbol: class ViewGroup
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/LayoutFactory.java:245: error: cannot find symbol
    private Options parseOptions(JSONObject jsonOptions) {
                                 ^
  symbol:   class JSONObject
  location: class LayoutFactory
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:61: error: cannot find symbol
    private final Activity activity;
                  ^
  symbol:   class Activity
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/child/ChildController.java:5: error: package android.os does not exist
import android.os.Build;
                 ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/child/ChildController.java:6: error: package android.util does not exist
import android.util.Log;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/child/ChildController.java:9: error: package android.view does not exist
import android.view.WindowInsets;
                   ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/parent/ParentController.java:29: error: cannot find symbol
public abstract class ParentController<T extends ViewGroup> extends ChildController<T> {
                                                 ^
  symbol: class ViewGroup
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/child/ChildController.java:27: error: cannot find symbol
public abstract class ChildController<T extends ViewGroup> extends ViewController<T> {
                                                ^
  symbol: class ViewGroup
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:67: error: cannot find symbol
    private ParentController<? extends ViewGroup> parentController;
                                       ^
  symbol:   class ViewGroup
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:84: error: cannot find symbol
    public ViewController(Activity activity, String id, YellowBoxDelegate yellowBoxDelegate, Options initialOptions, ViewControllerOverlay overlay) {
                          ^
  symbol:   class Activity
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:132: error: cannot find symbol
    public void addOverlay(View v, ViewGroup.LayoutParams layoutParams) {
                           ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:132: error: package ViewGroup does not exist
    public void addOverlay(View v, ViewGroup.LayoutParams layoutParams) {
                                            ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:136: error: cannot find symbol
    public void removeOverlay(View view) {
                              ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:177: error: cannot find symbol
    public Activity getActivity() {
           ^
  symbol:   class Activity
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:181: error: cannot find symbol
    public void performOnView(Func1<View> task) {
                                    ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/utils/CoordinatorLayoutUtils.java:9: error: static import only from classes and interfaces
import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/views/stack/StackLayout.java:3: error: package android.annotation does not exist
import android.annotation.SuppressLint;
                         ^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/views/stack/StackLayout.java:15: error: static import only from classes and interfaces
import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
^
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/views/stack/StackLayout.java:17: error: cannot find symbol
@SuppressLint("ViewConstructor")
 ^
  symbol: class SuppressLint
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:227: error: cannot find symbol
    public void attachView(ViewGroup parent, int index) {
                           ^
  symbol:   class ViewGroup
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:246: error: cannot find symbol
    public ViewController findController(View child) {
                                         ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:325: error: cannot find symbol
    public void onConfigurationChanged(Configuration newConfig) {
                                       ^
  symbol:   class Configuration
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:333: error: cannot find symbol
    public void onChildViewAdded(View parent, View child) {
                                 ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:333: error: cannot find symbol
    public void onChildViewAdded(View parent, View child) {
                                              ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:338: error: cannot find symbol
    public void onChildViewRemoved(View view, View view1) {
                                   ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
/Users/personal.devuser/Development/subfol/project-foldr/node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/viewcontroller/ViewController.java:338: error: cannot find symbol
    public void onChildViewRemoved(View view, View view1) {
                                              ^
  symbol:   class View
  location: class ViewController<T>
  where T is a type-variable:
    T declared in class ViewController
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
100 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-navigation:compileReactNative63DebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

build.gradle
```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    ext {
        kotlinVersion = "1.5.31" // To change the default react-native-gesture-handler Kotlin plugin version
        RNNKotlinVersion = kotlinVersion
        buildToolsVersion = "30.0.3"
        minSdkVersion = 21
        compileSdkVersion = 33
        targetSdkVersion = 33
        playServicesVersion = "17.0.0"
        androidMapsUtilsVersion = "2.0.3"
        ndkVersion = "20.1.5948944"
        androidXCore = "1.6.0"
    }
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
        classpath("com.android.tools.build:gradle:4.2.2")
        classpath 'com.google.gms:google-services:4.3.10'
        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.8.1'
        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        mavenLocal()
        maven {
            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
            url("$rootDir/../node_modules/react-native/android")
        }
        maven {
            // Android JSC is installed from npm
            url("$rootDir/../node_modules/jsc-android/dist")
        }

        google()
        jcenter()
        maven { url 'https://www.jitpack.io' }
    }
}
```

--- a/lib/android/app/build.gradle
+++ b/lib/android/app/build.gradle
@@ -65,11 +65,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_11
+        jvmTarget = JavaVersion.VERSION_1_8
     }
 
     flavorDimensions "RNN.reactNativeVersion"

--- a/playground/android/app/build.gradle
+++ b/playground/android/app/build.gradle
@@ -20,8 +20,8 @@ android {
     ndkVersion "20.1.5948944"
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
     }
 
     defaultConfig {

